src/main/resources/hibernate.cfg.xml

```diff
@@ -0,0 +1,15 @@
+<hibernate-configuration>
+    <session-factory>
+        <property name="hibernate.dialect">org.hibernate.dialect.MySQLDialect</property>
+        <property name="connection.driver_class">com.mysql.cj.jdbc.Driver</property>
+        <property name="connection.url">jdbc:mysql://localhost:3306/hibernate_db?useSSL=false</property>
+        <property name="connection.username">admin</property>
+        <property name="connection.password">admin</property>
+        <property name="show_sql">true</property>
+        <property name="hbm2ddl.auto">create-drop</property>
+
+        <mapping class="mate.academy.hibernate.relations.model.Movie" />
+        <mapping class="mate.academy.hibernate.relations.model.Actor" />
+        <mapping class="mate.academy.hibernate.relations.model.Country" />
+    </session-factory>
+</hibernate-configuration>
```

Comment:
```suggestion
</hibernate-configuration>

```
don't forget to leave last line empty
![image](https://github.com/mate-academy/hibernate-relations-hw/assets/49767049/a5d7c10b-84cf-4ea2-9158-113bae597f62)